client/state/global-sidebar/selectors.ts

```diff
@@ -20,7 +23,11 @@ export const getShouldShowGlobalSidebar = (
 		sectionGroup === 'me' ||
```

Comment:
nit: you can take the eslint comment off line 20 now 😄

Reply:
Done!